content/tutorials/mushr_unity_sim.md

```diff
@@ -0,0 +1,109 @@
+---
+title: "Using unity to simulate MuSHR"
+date: 2021-05-17T17:51:18+05:30
+summary: "Using a unity-based simulator in place of the default state prediction system that comes with mushr_base"
+difficulty: "Intermediate"
+duration: 0
+featured: false  # whether this is listed at / (must also be top 6 by weight). 
+active: true     # whether this is listed at /tutorials/
+draft: true      # whether Hugo considers this a draft
+weight: 3        # 2 = intro tutorial 3 = anything else
+---
+
+<h2> By: <a href=https://www.sidharthtalia.com/>Sidharth Talia</a></h2>
+
+<!-- Header figure required! -->
+<br>
+{{< figure src="/tutorials/mushr_unity_sim/thumbnail.gif" width="800" >}} <br>                           
+<br>
+
+### Introduction
+The default state prediction system [racecar_state.py](https://github.com/prl-mushr/mushr_base/blob/master/mushr_base/src/racecar_state.py) works well for low-speed situations where the wheel-slippage and body roll aren't significant. However, once you get to higher speeds (3-4 m/s and above), you want to be able to simulate those things as well so that you know your system is robust to slippage, body roll and so on. For this purpose, we use a unity-based simulator to simulate the state of the car with higher physics fidelity. The system provided by the [mushr_unity_sim](https://github.com/naughtyStark/mushr_unity_sim.git) is a drop in replacement for the default state predictor.
+
+<br>
+{{< figure src="/tutorials/mushr_unity_sim/thumbnail.png" width="800" >}} <br>                           
```

Comment:
oh super cool pic!

Reply:
maybe this is better suited for the nhttc tutorial though?

Reply:
you're probably right. I wanted to have a "mean looking" line-up somewhere. I can certainly put this in the nhttc tutorial. I'll put a different gif here.